include/pybind11_json/pybind11_json.hpp

```diff
@@ -129,4 +129,48 @@ namespace nlohmann
 
 }
 
+namespace pybind11
+{
+    namespace detail
+    {
+        template <> struct type_caster<nlohmann::json>
+        {
+        public:
+            /**
+             * This macro establishes the name 'user_type' in
```

Comment:
The comment is not in sync with the code, is it?
Shouldn't `user_type` be `nlohmann_json` here?